README.md

```diff
@@ -38,6 +35,10 @@ Prisma is a **next-generation ORM** that consists of these tools:
 
 Prisma Client can be used in _any_ Node.js or TypeScript backend application (including serverless applications and microservices). This can be a [REST API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/rest), a [GraphQL API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/graphql), a gRPC API, or anything else that needs a database.
 
+**The Prisma ORM can also further be extended with these Prisma solutions:**
+- [Prisma Accelerate](https://prisma.io/docs/data-platform/accelerate/what-is-accelerate?utm_source=github&utm_medium=org-readme): Global database cache with scalable connection pooling
+- [Prisma Pulse](https://www.prisma.io/docs/data-platform/pulse/what-is-pulse?utm_source=github&utm_medium=org-readme): Real-time database events with type-safe subscriptions
```

Comment:
Do these `utm` links allow us to track the number of clicks these get reliably?

Reply:
Yes, but this reminds me that I need to adjust them

Reply:
Actually a pris.ly here could work better if you want to know the absolute number of clicks. And we could still assign UTM parameters.